tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -646,3 +646,8 @@ test_i2c_as5600: as5600@54 {
 	compatible = "ams,as5600";
 	reg = <0x54>;
 };
+
+test_i2c_bh1750: bh1750@23 {
+	compatible = "rohm,bh1750";
+	reg = <0x23>;
+};
```

Comment:
Please set the address to 0x55 to make it sequential. Note this is a build test that will never run on real hardware

Reply:
Ok, I see it. However, the previous one `test_i2c_as5600` is incorrect and should be `64` to follow the correct numbering. Thus, I propose to set the `test_i2c_bh1750` address to 0x65.